docs/theming/developers.md

```diff
@@ -377,7 +386,7 @@ And your document CSS sets the desired color context:
 ```
 
 And you would declare the import a shared stylesheet which activates the theming system,
```

Comment:
`which` → `that`

Reply:
[8ae3170](https://github.com/RedHat-UX/red-hat-design-system/pull/1937/commits/8ae31707ea11f29ae967709bb95afebae9820f44)